db/schema.rb

```diff
@@ -123,8 +123,8 @@
   create_table "distributors_shipping_methods", :force => true do |t|
     t.integer  "distributor_id"
     t.integer  "shipping_method_id"
-    t.datetime "created_at",         :null => false
-    t.datetime "updated_at",         :null => false
+    t.datetime "created_at"
+    t.datetime "updated_at"
```

Comment:
Sorry, I didn't remember this PR quite well. You do have a migration in this PR. `rake db:migrate:up VERSION=20170512115519` should work. Try to `down` it and then `up` again the specific migration.